Add installation section to coverage docs (#2473)
foundry-rs/starknet-foundry: docs/src/testing/coverage.md

Coverage reporting allows developers to gain comprehensive insights into how their code is executed.
With `cairo-coverage`, you can generate a coverage report that can later be analyzed for detailed coverage statistics.

## Installation and usage

In order to run coverage report with `cairo-coverage` you need to install it first. 

> Please refer to the instructions in the readme on how to do this:  

Reviewer: nit:
```suggestion
Please refer to the instructions provided in the README for guidance:
```
